src/main/external-resources/renderers/DefaultRenderer.conf

```diff
@@ -621,8 +621,10 @@ AutoExifRotate =
 #    mkv      Matroska Video                                video/x-matroska
 #    mlp      Meridian Lossless Packing                     audio/vnd.dolby.mlp
 #    mov      QuickTime File Format                         video/quicktime
+#    mp2      MPEG Audio Layer II                           audio/mpeg
```

Comment:
@Sami32 I've changed the standard to ```audio/vnd.dolby.dd-raw``` since that is the correct mime-type according to the DLNA standard. If any renderers have a problem with it, it should be overridden on the renderer.